org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/WollokLauncherInterpreterEvaluator.xtend

```diff
@@ -42,25 +45,30 @@ class WollokLauncherInterpreterEvaluator extends WollokInterpreterEvaluator {
 	
 	override dispatch evaluate(WTest test) {
 		try {
-			wollokTestsReporter.testStart(test)
+			//wollokTestsReporter.testStart(test)
```

Comment:
This hook is not needed anymore ? What was it doing on testStart ? Can we say we dont' need this code anymore ?